packages/config/galaxy/__init__.py

```diff
@@ -0,0 +1 @@
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)  # type: ignore[has-type]
```

Comment:
Files in `packages` are not formatted with `make format`, so better to get it right before merging forward.

```suggestion
__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore[has-type]
```